### ISSUE TITLE
Why not change to a better  class name

### ISSUE THREAD

**Issue by suxin1995** [lukepighetti/fluro#185]
Why not change to a better  class name.

"Router" in the package  is same as official.

![image](https://user-images.githubusercontent.com/32704389/96073713-90810a00-0ed9-11eb-940a-bbfff5bacd0b.png)

so i have to use 'as ..' to solve
```
import 'package:flutter/material.dart' as mater;  

handlerFunc: (mater.BuildContext context,  Map<String, List<String>> params){
        return mainPage();
    });
```
it just suggestion

**Comment by lukepighetti** [comment]
Resolved by https://github.com/lukepighetti/fluro/commit/a96fb1393c16bbbd0022b602c85c194eaa144f60

**Comment by suxin1995** [comment]
> Resolved by [a96fb13](https://github.com/lukepighetti/fluro/commit/a96fb1393c16bbbd0022b602c85c194eaa144f60)

thanks! you are so cool
